tests/backup/backup_azure_credential_test.go

```diff
@@ -0,0 +1,216 @@
+package tests
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	"github.com/pborman/uuid"
+	api "github.com/portworx/px-backup-api/pkg/apis/v1"
+	"github.com/portworx/torpedo/drivers/backup"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
+	"golang.org/x/sync/errgroup"
+)
+
+// This testcase creates Azure cloud account with mandatory and non-mandatory fields and take backup & restore
+var _ = Describe("{AzureCloudAccountCreationWithMandatoryAndNonMandatoryFields}", Label(TestCaseLabelsMap[AzureCloudAccountCreationWithMandatoryAndNonMandatoryFields]...), func() {
+
+	var (
+		credUidWithAllFields                                   string
+		sourceClusterUid                                       string
+		backupNameWithBkpLocationHavingMandatoryParameters     string
+		restoreNameWithBkpLocationHavingMandatoryParameters    string
+		backupNameWithBkpLocationHavingAllParameters           string
+		restoreNameWithBkpLocationHavingAllParameters          string
+		backupNameWithBkpLocationHavingOnlyMandatoryParameters string
+		restoreNameWithBkpLocationHavingOnlyMandatoryFields    string
+		credUidWithMandatoryFields                             string
+		azureCredNameWithAllFields                             string
+		azureCredNameWithMandatoryFields                       string
+		azureBackupLocationNameWithAllFields                   string
+		azureBackupLocationNameWithMandatoryFields             string
+		backupLocationAllFieldsUID                             string
+		backupLocationMandatoryFieldsUID                       string
+		appNamespaces                                          []string
+		controlChannel                                         chan string
+		errorGroup                                             *errgroup.Group
+		azureConfigFields                                      *api.AzureConfig
+		scheduledAppContexts                                   []*scheduler.Context
+		contexts                                               []*scheduler.Context
+		appContexts                                            []*scheduler.Context
+	)
+
+	backupLocationMap1 := make(map[string]string)
+	cloudCredentialMap1 := make(map[string]string)
+	backupLocationMap2 := make(map[string]string)
+	cloudCredentialMap2 := make(map[string]string)
```

Comment:
Please move this inside `JustBeforeEach`

Reply:
done